main.py

```diff
@@ -0,0 +1,75 @@
+import soundfile as sf
```

Comment:
Do we really want this file to be shipped into the repo?

Reply:
corrected